tests/ui/specialization/ctfe/default-assoc-const.rs

```diff
@@ -0,0 +1,21 @@
+// ICE fixed by #119821.
+//
+// See that PR for more details.
+//
+// check-pass
```

Comment:
doesn't this pass rn? https://play.rust-lang.org/?version=nightly&mode=debug&edition=2021&gist=7dba8d9aa1ced839c563e298a595e431

Reply:
> // ICE fixed by #119821.

that's why I commented this. It did not ICE before :sweat_smile:

Reply:
Whoops, I updated the other tests' comments, but forgot this one